test/test_helper.rb

```diff
@@ -38,6 +38,19 @@
 require 'logger'
 
 
+require 'simplecov'
+# Note: minimum coverage config can also maybe happen outside, and before the start block.
+SimpleCov.start 'rails' do
+  # fail unit tests if total coverage dips below acceptable limit
+  SimpleCov.minimum_coverage 7
+  # fail unit tests if any file's individual coverage dips below acceptable limit
+  SimpleCov.minimum_coverage_by_file 0
+  add_filter '/bin/'
+  add_filter '/db/'
+  add_filter '/test/' # for minitest
+end
+
+
```

Comment:
This is code from a previous PR, that somehow got deleted by github in the merge.